src/corona/emissivity.jl

```diff
@@ -1,10 +1,12 @@
 """
-    source_to_disc_emissivity(m, N, A, x, g; Γ = 2)
+    source_to_disc_emissivity(m, N, A, x, g)
 
 Compute the emissivity of a disc element with (proper) area `A` at coordinates `x` with metric
-`m`. Since the emissivity is dependent on the incident flux, the photon (geodesic) count `N` must
-be specified, along with the ratio of energies `g` (computed with [`energy_ratio`](@ref)) and photon index
-`Γ`.
+`m` and coronal spectrum 'spec'. Since the emissivity is dependent on the incident flux, the photon (geodesic) count `N` must
+be specified, along with the ratio of energies `g` (computed with [`energy_ratio`](@ref)) and the spectrum 'spec'. 
+Spectrum for the power law simply returns the photon index 'Γ'. 'I' stores the 'coronal_spectrum' which is a function 
+of 'spectrum' and 'g' since it is defined as 'g^Γ' so passing the spectrum as an argument allows their further modification.
```

Comment:
This is good, but should be in the docstring for `PowerLawSpectrum`, since it is specific to that spectrum.